experiment_manager.py

```diff
@@ -351,18 +365,26 @@ def sweep(self, config):
         outer_variables = Config(*variable_config.param_list[:-1])  # "outer" means all but the inner-most loop
         # N-dimensional loop with itertools.product: # (actually N-1 )
 
+        counter = 1
+        num_jobs = 1
+        for var in outer_variables.param_list:
+            num_jobs=num_jobs*len(var.value)
 
         for indices, vals in enumerated_product(*outer_variables.get_values()):
             # update parameters to current values:
             for i, param in enumerate(outer_variables.param_list):
                 curr_config.set_parameter(name=param.name, value=vals[i])
 
-            print("running...")
+            print(f"running job {counter} out of {num_jobs}...")
             # do 1D sweep on the tracing parameter:
-            print(curr_config.param_list) #TODO only print sweep parameters
+            print('current configuration:')
+            for par in outer_variables.param_list:
+                print(getattr(curr_config,par.name))
```

Comment:
This should probably be a method of `Config`

Reply:
but this is more specific than that since outer_variables and curr_config are not the same
here I want to print other current scalar values of the looped parameters

but anyway a method Config.get_names_list() is a good idea and I added it